proof/refine/ARM_HYP/CNodeInv_R.thy

```diff
@@ -4129,6 +4147,75 @@ lemmas ep_simps =
   isEPsrc isEPbadge_src isNTFNsrc isNTFNbadge_src
   isEPdest isEPbadge_dest isNTFNdest isNTFNbadge_dest
 
+(* FIXME arch-split: arch dependent part of mdb_swap proofs, no arch_global_naming because of locale *)
```

Comment:
not sure I follow, which locale?

Reply:
This whole block is inside `context mdb_swap`